### DISCUSSION TITLE
Connecting to the RCT app and RCT Protokol Warning

### DISCUSSION THREAD

**Discussion by waldayer1** [weltenwort/home-assistant-rct-power-integration#401] [Q&A]
Nice Sunday

The Integration runs without Problems for about 8 Months.
Today I tryed to connect me with the RCT Power app (right ID added), it didnt find my WR, so i removed the Intergration, (My thougt: Maybe the WR can only report to one) but it still didnt wok.
I installed the integration again, it worked, all parameters were on board and on the dashboards, so far so good.
Then I shut down the Raspi on Homeassistant, the app still didnt work.
I resarted the Raspi without problem, everything fine. The app didnt find my WR as before.
Now Question 1:
Is it a problem to use homeassistant and the RCT app at the same time? Might help a modbusproxy?
Question 2: during searching around in my homeassistant i found in the Logs (Protokolle) these warnigs.
I checked, they have been reported already. Any reason to worry? I thnik thats not a Problem of my homeassistant, or am i wrong?
The restart was at this time.
The last days i moved from SD to mnev on Raspi 5, several starts with te same warnig.

Logger: homeassistant.helpers.fram
Looking in the Logs I found the e
Quelle: helpers/frame.py:234
Erstmals aufgetreten: 12:12:33 (1 Vorkommnisse)
Zuletzt protokolliert: 12:12:33

Detected code that calls async_forward_entry_setup for integration, rct_power with title: RCT Power Inverter at 192.168.50.187:8899 and entry_id: 01JEJWCXF1EXDWGHY65HRGQYMF, which is deprecated, await async_forward_entry_setups instead. This will stop working in Home Assistant 2025.6, please report this issueLogger: homeassistant.helpers.frame

Quelle: helpers/frame.py:324
Erstmals aufgetreten: 12:12:33 (1 Vorkommnisse)
Zuletzt protokolliert: 12:12:33

Detected that custom integration 'rct_power' calls `async_add_job`, which should be reviewed against https://developers.home-assistant.io/blog/2024/03/13/deprecate_add_run_job for replacement options at custom_components/rct_power/__init__.py, line 122: hass.async_add_job( # type: ignore. This will stop working in Home Assistant 2025.4, please create a bug report at https://github.com/weltenwort/home-assistant-rct-power-integration/issues
Logger: homeassistant.core
Quelle: helpers/deprecation.py:222
Erstmals aufgetreten: 12:12:27 (1 Vorkommnisse)
Zuletzt protokolliert: 12:12:27

Config was used from rct_power, this is a deprecated alias which will be removed in HA Core 2025.11. Use homeassistant.core_config.Config instead, please report it to the author of the 'rct_power' custom integration

Thanks for reply

**Comment by dahadd** [comment]
HA and RCT POWER APP can run together.

**Reply by waldayer1** [reply]
Thanks
so i have look on whats wrong. 
Might it be a problem, that the integration  "blocks" the app bay using the port of the WR "The port of the inverter's API, defaults to 8899" 
We`ll see
good night
